irsdb/metadata/management/commands/generate_schemas_from_metadata.py

```diff
@@ -116,7 +117,6 @@ def write_sked(self, schedule):
 
             variables_in_this_part = Variable.objects.filter(
                 parent_sked_part=form_part.parent_sked_part,
-                version_end__in=['','2016', '2017', '2018'],
```

Comment:
this is what was leading to the `TypeError: pf_part_xii() got unexpected keyword arguments: 'PctSct4940OrgNtInvstIncmAmt', 'AdjstdQlfyngDstrAmt'`